executor/src/witgen/jit/single_step_processor.rs

```diff
@@ -237,7 +233,7 @@ if (VM::instr_add[0] == 1) {
         col fixed INSTR_ADD = [0, 1] + [0]*;
         col fixed INSTR_MUL = [1, 0] + [1]*;
 
-        pc' = pc + 1;
```

Comment:
This change was needed because otherwise, the system is not solvable. Do a case analysis of the allowed values for `pc` and `pc'` and you will see why.